js-loops_forin-select-nav/js/index.js

```diff
@@ -39,4 +46,13 @@ navElement.append(ul);
 
 // --v-- write/change code here --v--
 
+for (const key in nav) {
+  const li = document.createElement("li");
```

Comment:
li here is not necessary. you can directly use <a> inside a ul and it works as expected. 
[reference](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_navbar_icon)